tests/ItemsTest.php

```diff
@@ -128,8 +136,15 @@ public function testUpdateItem()
     {
         $item = $this->addItem();
 
+        Event::fake();
+
         $this->laracart->updateItem($item->getHash(), 'qty', 4);
 
+        Event::assertDispatched('laracart.updateItem', function ($e, $eventItem) use ($item) {
+            return $eventItem['item'] === $item && $eventItem['newHash'] === $item->getHash();
+        });
+        Event::assertDispatched('laracart.updateItem', 1);
```

Comment:
This is causing errors, did you mean to add this line?

Reply:
Event::assertDispatched('laracart.updateItem', 1);

Reply:
Yes this rule is checking if the event is fired once. I was checking this as we speak. I think this rule is supported from Laravel 5.6+ and your package is requiring Laravel 5.4.

Reply:
Founded another solution. :)  I'll commit it in a few seconds.